### PR TITLE
Feat/return function

### PR DESCRIPTION
**Pull request by thienanhvu**

### CODE REVIEW

File: pho/src/components/Pomodoro/Pomodoro.tsx [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+const getWindowDimensions = () => {
+  const { innerWidth: width, innerHeight: height } = window;
+  return {
+    width,
+    height,
+  };
+};
+
+const pomodoroWidthPixels = 250;
+const toolbarRightPosition = 62;
```
Comment (minhnguyenphamquoc) [inline pho/src/components/Pomodoro/Pomodoro.tsx:12]: Please put `getWindowDimensions` function inside `lib` folder as. I think it should be considered as utility function. @thienanhvu

File: pho/src/components/RegisterContainer/Register.tsx [hunk 2]
```diff
@@ -63,6 +66,15 @@ export default function Register({ onOpenLogin }: RegisterProps) {
...
 
       await registerViaAPI(username, salt, hashedPassword);
+      await toast({
+        title: 'Account created successfully.',
+        description:
+          "We've created your account for you. Please sign in to continue",
+        status: 'success',
+        duration: 9000,
+        isClosable: true,
+      });
```
Comment (minhnguyenphamquoc) [inline pho/src/components/RegisterContainer/Register.tsx:76]: `9000` milliseconds is quite long for a pop-up to be shown in the main screen. Can you adjust the time to be around 3000ms-4000ms? @thienanhvu

**Review by minhnguyenphamquoc — COMMENTED** [review]
Overall it is good but need a little adjustment to some functionality.

**Review by chaule-mck — APPROVED** [review]

File: pho/src/components/Pomodoro/Pomodoro.tsx (hunk 1 above)
Comment (thienanhvu) [reply pho/src/components/Pomodoro/Pomodoro.tsx:12]: Resolved! Thank you

File: pho/src/components/RegisterContainer/Register.tsx (hunk 2 above)
Comment (thienanhvu) [reply pho/src/components/RegisterContainer/Register.tsx:76]: Resolved! Thank you

**Review by richardtlp — APPROVED** [review]
The function works

**Review by minhnguyenphamquoc — APPROVED** [review]
LGTM!

**Pull request merged**